src/gordon_gcp/plugins/janitor/gpubsub_publisher.py

```diff
@@ -236,6 +247,6 @@ def publish(self, message):
             try:
                 await self.publish(change_message)
             except Exception as e:  # todo
-                logging.error(f'foo: {e}')
+                logging.error(f'Exception while trying to publish record: {e}')
```

Comment:
I'll update this message especially because I think 'record' is misleading, but I don't think there will be much confusion in practice since janitor log messages are isolated from core service, and I suspect the error being excepted here will contain Google Pubsub text.